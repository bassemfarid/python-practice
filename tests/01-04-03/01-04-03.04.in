55286.0
81988.0