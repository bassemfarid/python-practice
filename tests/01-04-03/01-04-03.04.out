2266394284.0